### PR TITLE
Handle records in syntax highlighting

### PR DESCRIPTION
**Pull request by 333fred**
Fixes https://github.com/OmniSharp/omnisharp-roslyn/issues/2048.

### CODE REVIEW

**Comment by 333fred** [comment]
@JoeRobich for review.

File: src/OmniSharp.Roslyn.CSharp/Services/SemanticHighlight/SemanticHighlightService.cs [hunk 1]
```diff
@@ -107,6 +107,7 @@ class ClassifiedResult
                 [ClassificationTypeNames.VerbatimStringLiteral] = SemanticHighlightClassification.VerbatimStringLiteral,
                 [ClassificationTypeNames.StringEscapeCharacter] = SemanticHighlightClassification.StringEscapeCharacter,
                 [ClassificationTypeNames.ClassName] = SemanticHighlightClassification.ClassName,
+                [ClassificationTypeNames.RecordName] = SemanticHighlightClassification.ClassName,
```
Comment (333fred) [inline src/OmniSharp.Roslyn.CSharp/Services/SemanticHighlight/SemanticHighlightService.cs:110]: The other option would be to map a separate classification for record names and have the clients deal with it. Which is fine, just need to know what to do.

Comment (filipw) [reply src/OmniSharp.Roslyn.CSharp/Services/SemanticHighlight/SemanticHighlightService.cs:110]: no, this is fine IMHO

**Review by filipw — APPROVED** [review]

File: src/OmniSharp.Roslyn.CSharp/Services/SemanticHighlight/SemanticHighlightService.cs (hunk 1 above)
Comment (333fred) [reply src/OmniSharp.Roslyn.CSharp/Services/SemanticHighlight/SemanticHighlightService.cs:110]: Well, the only concern is that we'll need to do something different when struct records come out, but that's a concern for next year :).

**Pull request merged**